app/javascript/onboarding/components/FollowTags.jsx

```diff
@@ -135,48 +131,37 @@ export class FollowTags extends Component {
                 return (
                   <div
                     className={`onboarding-tags__item ${
-                      selected && 'onboarding-tags__item--selected'
+                      selected ? 'onboarding-tags__item--selected' : ''
                     }`}
-                    style={{
-                      boxShadow: selected
-                        ? `inset 0 0 0 100px ${tag.bg_color_hex}`
-                        : `inset 0 0 0 0px ${tag.bg_color_hex}`,
-                      color: selected ? tag.text_color_hex : '',
-                    }}
+                    aria-label={`Follow ${tag.name}`}
                     key={tag.id}
+                    onClick={() => this.handleClick(tag)}
```

Comment:
@maestromac 
I am a bit confused about Code-coverage.
It says that like #L138 and #L142 are not covered in tests but I have written tests in `FollowTags.test.jsx` which covers both of these. The test names are following:
`should call handleClick when enter key is pressed`
`should call handleClick when space key is pressed`
`should update the status and count when you follow a tag`

Any idea what I am missing here?

Reply:
For line 138:
You have tests that are testing enter and space key presses, but the code itself says `onClick`. The missing coverage is correct because you didn't write a test that simulate clicks.

For line 142:
I added a console.log to see why it wasn't showing up.
```
                    onKeyPress={(event) => {
                      // Trigger for enter (13) and space (32) keys
                      console.log('event.which', event.which);
                      if (event.which === 13 || event.which === 32) {
                        this.handleClick(tag);
                      }
                    }}
```
The event I’m seeing when running the test locally is `0`. That handle click never ran.

Reply:
Thanks a lot @maestromac for debugging this. On further investigation I found that there were two major culprits:
1. `event.which` in this `which` is deprecated instead `keyCode` is more correctly now, but the bigger problem was `fireEvent` key trigger code needed more information.
2. I was using `queryByText` without any `wait` which I updated it to use getByText` and using `wait` which was more accurate code.

 Both of these fixes the tests and coverage also passes. Thanks :-)